lib/rubocop/cop/lint/meterpreter_commands_dependencies.rb

```diff
@@ -0,0 +1,260 @@
+                "#{commands_whitespace}'Commands' => %w[" \
+                "\n#{array_content_whitespace}#{@identified_commands.join("\n#{array_content_whitespace}")}\n" \
+                "#{commands_whitespace}]\n" \
+                "#{meterpreter_whitespace}}"
+
+              corrector.replace(meterpreter_hash_node, new_hash)
+
+            # Handles scenario when we have a compats hash, but no meterpreter hash
+            # and compats array present within the array
+            elsif @compat_node && @meterpreter_node.nil? && @command_node.nil?
+              compat_hash_node = @compat_node.children[1]
+
+              # White spacing handling based of node offsets
+              compat_whitespace = offset(@compat_node)
+              meterpreter_whitespace = compat_whitespace + "  "
+              commands_whitespace = meterpreter_whitespace + "  "
+              array_content_whitespace = commands_whitespace + "  "
+
+              # Formatting to add missing commands node when the method has a compat node & meterpreter node present
+              new_hash =
+                "{\n" \
+                "#{meterpreter_whitespace}'Meterpreter' => {\n" \
+                "#{commands_whitespace}'Commands' => %w[" \
+                "\n#{array_content_whitespace}#{@identified_commands.join("\n#{array_content_whitespace}")}\n" \
+                "#{commands_whitespace}]\n" \
+                "#{meterpreter_whitespace}}\n" \
+                "#{compat_whitespace}}"
+
+              corrector.replace(compat_hash_node, new_hash)
+
+            elsif @compat_node.nil? && @meterpreter_node.nil? && @command_node.nil? && !@initialize_node.nil?
+              # require "pry"; binding.pry
+              # White spacing handling based of node offsets
+              compat_whitespace = offset(@end_of_info_node)
+              meterpreter_whitespace = compat_whitespace + "  "
+              commands_whitespace = meterpreter_whitespace + "  "
+              array_content_whitespace = commands_whitespace + "  "
+
+              # Formatting to add missing commands node when the method has a compat node & meterpreter node present
+              test_new_hash =
+                ",\n#{compat_whitespace}'Compat' => {\n" \
+                "#{meterpreter_whitespace}'Meterpreter' => {\n" \
+                "#{commands_whitespace}'Commands' => %w[" \
+                "\n#{array_content_whitespace}#{@identified_commands.join("\n#{array_content_whitespace}")}\n" \
+                "#{commands_whitespace}]\n" \
+                "#{meterpreter_whitespace}}\n" \
+                "#{compat_whitespace}}"
+
+              corrector.insert_after(@end_of_info_node, test_new_hash)
+
+            elsif @compat_node.nil? && @meterpreter_node.nil? && @command_node.nil? && @initialize_node.nil?
+              # White spacing handling based of node offsets
+              def_whitespace = offset(@node_after_initialize)
+              super_whitespace = def_whitespace + "  "
```

Comment:
This should be using `indent(...)`

I think things weren't working as expected, as it reads the required whitespace from its configuration:
```ruby
# gems/rubocop-1.16.0/lib/rubocop/cop/mixin/alignment.rb
def indentation(node)
    offset(node) + (SPACE * configured_indentation_width)
end
```

You'll want to update the tests to have the configured indentation width to work:
```rspec
  let(:config) do
    RuboCop::Config.new(
      'Layout/IndentationWidth' => {
        'Width' => indentation_width
      })
  end
  let(:indentation_width) { 2 }
```